sdks/python/apache_beam/io/gcp/bigquery_read_it_test.py

```diff
@@ -96,6 +96,29 @@ def create_table(self, tablename):
     self.bigquery_client.insert_rows(
         self.project, self.dataset_id, tablename, table_data)
 
+  def create_table_non_ascii(self, tablename):
+    table_schema = bigquery.TableSchema()
+    table_field = bigquery.TableFieldSchema()
+    table_field.name = 'str'
+    table_field.type = 'STRING'
+    table_schema.fields.append(table_field)
+    table = bigquery.Table(
+        tableReference=bigquery.TableReference(
+            projectId=self.project,
+            datasetId=self.dataset_id,
+            tableId=tablename),
+        schema=table_schema)
+    request = bigquery.BigqueryTablesInsertRequest(
+        projectId=self.project, datasetId=self.dataset_id, table=table)
+    self.bigquery_client.client.tables.Insert(request)
+    table_data = [
+        {'str': '你好'},
```

Comment:
I'd suggest we combine this scenario with standard read scenario to avoid starting extra pipeline and reduce flakiness. WDYT?